include/vapor/SliceRenderer.h

```diff
@@ -36,52 +37,73 @@ class RENDER_API SliceRenderer : public Renderer {
         int                 compressionLevel;
         int                 textureSampleRate;
         int                 orientation;
+        double              xRotation;
```

Comment:
A miner nit, but other vector arguments are passed in vector form why not be consistent? Instead of XRotation, yRotation, zRotation, why not:

`std::vector<double> rotation`

Reply:
Copied from above:

> Single values (vs vectors) allow use of the low level interface. No validation needs to be done for rotation on the X, Y, or Z axis. So all we need is a single parameter for each rotation, and an LLI widget like what we have at line 29 of SliceEventRouter.cpp.
> 
> This follows the policy we wrote on PWidgets a while back.